### DISCUSSION TITLE
2005/08/13/introduce-my-hometown

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#47] [Announcements]
# 2005/08/13/introduce-my-hometown

I lived in a small town in the northeast of Inner Mongolia province, named Jagedaqi (加格达奇), before I entered the elementary school.  It’s located at the foot of Xing’an Mountains.  Although I had lived in there so many years, I was too young to travel around.  I could hardly remember what the town looks like until one summer a couple years ago, when I got a chance to visit it again.

https://blog.qizhen.xyz/2005/08/13/introduce-my-hometown

**Comment by ruanqizhen** [comment]
zoe
2009年04月7日 12:14 编辑
我学的专业好像跟你们的不搭架，不过没关系，你这篇Introduce My Hometown确实写得不错，我要借鉴一下了，不好意思，我要参加口语考试，所以。。。。。。呵呵呵。。。。。。

回复
Terry
2008年10月12日 13:45 编辑
Its really a nice place and admiring this miracle of your childhood.

回复
幽灵公主
2008年01月8日 16:45 编辑
it\’s a very interesting place
i love the pretty snowflake but i scare the terrible accident
you have got a nice childhood

回复
Qizhen
2005年08月22日 21:23 编辑
我们国家因为政治的原因，差不多是无原则的支持达尔文的进化论。对于稍微有利于进化论的证据，甚至是未经证实的证据都夸大宣传。即便如此，能作为过渡类型的化石也只有始祖鸟等极少的几个。相对于其他的成千上万的物种的化石显得太微乎其微了。实际上大多数化石展示的都是突然出现的某种有机体，这些有机体一旦出现，基本上就不再变了，哪怕过了几百万年。在30亿年前到5亿年前之间的化石全部都是单细胞生物的。然而，在5亿年前，似乎一夜之间，各式各样的多细胞生物就出现在了海洋里。所有可以找到的化石都属于这些物种，根本没有介于他们之间的过渡类型。动物界有十个门，脊索动物门是哪来的？是从原生动物门进化来的，还是从棘皮动物门进化来的？像类似这些问题，在化石里一点线索都找不到。

回复
JING
2005年08月19日 01:34 编辑
一点肤浅的看法:站长所说的基因突变这个名词我很熟悉,但没从生物进化的角度去看过,为什么基因突变的理论会导出"生物进化的进程应该是一个均匀连续的过程",基因的突变应该是随机的,自发的,经突变而具有有利基因的生物被环境选择而得以继续进化,同时环境的变化从一个相当长的时间来看,可以认为是均匀和连续的,所以我认为从某种角度来看,这个进程是均匀和连续的.至于化石,"然而，在出土的化石中，基本都是物种分明的，很难找到介于不同物种的中间过渡类型",这样说,是不是不太确切.我们在高中的生物课本里,就已经学到了很多例子了,也许站长对于"物种"的划分更为细致一些.

回复
卡
2005年08月16日 08:05 编辑
也来看看

回复
米拉
2005年08月15日 23:12 编辑
感觉很深奥 我很向往上海

回复
BYYue
2005年08月15日 22:52 编辑
回踩，你得space内容真够充实，有空常去我那看看

回复
Desmond
2005年08月15日 21:58 编辑
我也SH的…

回复
Desmond
2005年08月15日 21:57 编辑
我大学想学AI或者ROBOTICS,还是要到你这取取经啊…

回复
JING
2005年08月14日 00:01 编辑
使我记起高中时候在<科幻世界>上看过的一篇小说,讲的是计算机的什么程序之类的自己进化然后产生了人工智能的故事^_^

回复

**Comment by Chen-GISer** [comment]
加格达奇本地人原来认为自己是内蒙人，我一直还觉得加格达奇是黑龙江的哈哈

**Reply by ruanqizhen** [reply]
加格达奇这个地方确实比较特殊。建国后，黑龙江和内蒙古的边界经历过多次调整，它的归属也随之几经变动。目前它在区划上名义属于内蒙古，但实际是由黑龙江在管辖。虽然我已经很久没和那边联系了，但我估计，当地人在心理认同上还是会更偏向黑龙江一些。
